extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetSchemaReader.java

```diff
@@ -327,6 +328,9 @@ public Optional<Class<?>> visit(final LogicalTypeAnnotation.EnumLogicalTypeAnnot
             @Override
             public Optional<Class<?>> visit(
                     final LogicalTypeAnnotation.DecimalLogicalTypeAnnotation decimalLogicalType) {
+                if (decimalLogicalType.getPrecision() == 1 && decimalLogicalType.getScale() == 0) {
```

Comment:
Can we add a comment here that the 1 is a "marker" value that we ourselves set up before?
An unsuspecting reader (me, in the past, before getting an explanation) may ask themselves "why is it that the precision would be exactly 1?"